docs/en/developer/create-web-component/REACT.md

```diff
@@ -0,0 +1,59 @@
+# Creating a React web component
+
+> A JavaScript library for building user interfaces
+
+<https://reactjs.org>
+
+## Advantages
+
+* React has a large developer community
+
+## Trade offs
+
+* React does not officially support web components, and does not intend to
+* React DOM adds ~100kb to each component
+
+## Prerequisites
+
+* [Git][]
+* [Node.js][] 10.3+ (using [NVM][] is recommended)
+
+## Setup
+
+```sh
+# clone the repository if you haven't already
+git clone https://github.com/uPortal-contrib/uPortal-web-components
```

Comment:
should be git clone https://github.com/uPortal-contrib/uPortal-web-components.git

Reply:
Hey Julien,

What error is being shown cloning without `.git`?
And what `git --version` is being run?

Reply:
Sorry it's my fault I didn't see that I added a bad char into the command git, there isn't problem so !  
Should go home to rest :sleepy:

Reply:
No worries